javascript/ql/src/Security/CWE-598/SensitiveGetQuery.ql

```diff
@@ -0,0 +1,27 @@
+/**
+ * @name Sensitive data read from GET request
+ * @description Placing sensitive data in a GET request increases the risk of
+ *              the data being exposed to an attacker.
+ * @kind problem
+ * @problem.severity warning
+ * @security-severity 6.5
+ * @precision high
+ * @id js/sensitive-get-query
+ * @tags security
+ *       external/cwe/cwe-598
+ */
+
+import javascript
+
+from
+  Express::RouteSetup setup, Express::RouteHandler handler, Express::RequestInputAccess input,
```

Comment:
Does it cause many false positives to use `HTTP` instead of `Express` here?

Reply:
Nope, I haven't tried that.  

The implementation is specific to `Express` because that had an easy "this is a GET request" predicate.